releasenotes/notes/0.8/fixes_qiskit_1_2-6d60334235081088.yaml

```
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    Qiskit Experiments was updated to be compatible with Qiskit 1.2, 
+    including removing references to :meth:`QuantumCircuit._parameter_table`
```

I think we need to format this as code and not try to link to documentation since the problem was that we were using this undocumented method that was removed (so there won't be a documentation link).

```suggestion
    including removing references to ``QuantumCircuit._parameter_table``
```